ci/kokoro/windows/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+# ~~~
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+cmake_minimum_required(VERSION 3.5)
+project(google-cloud-cpp-spanner-super-build CXX C)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(GOOGLE_CLOUD_CPP_SPANNER_ROOT "${PROJECT_SOURCE_DIR}/../../..")
+list(APPEND CMAKE_MODULE_PATH "${GOOGLE_CLOUD_CPP_SPANNER_ROOT}/cmake")
+
+include(external/external-project-helpers)
+google_cloud_cpp_set_prefix_vars()
+
+include(ExternalProject)
+set(
+    GOOGLE_CLOUD_CPP_URL
+    "https://github.com/googleapis/google-cloud-cpp/archive/84772e182e5a82c9f839a9073cef88b9962c48cf.tar.gz"
+    )
+set(GOOGLE_CLOUD_CPP_SHA256
+    "3bdf9266cd16c935f2c62bf6d151df8a15a307d612e966be0f340398785b9d31")
+google_cloud_cpp_set_prefix_vars()
+
+ExternalProject_Add(
+    # Use very short names for the project. On Windows the path limit is about
+    # 250 characters. The project name and prefix become part of the path, and
+    # in the case of googleapis (a sub-project of `google-cloud-cpp`) using
+    # full project names creates target files that exceed Windows limit.
+    g-c-cpp
+    EXCLUDE_FROM_ALL ON
+    PREFIX "${CMAKE_BINARY_DIR}/e/g-c-cpp"
```

Comment:
Do we need a separate CMake file to use shorter paths?  Any other reason?  If it is just for the shorter paths then I think I prefer to first change the current CMake file in ci/super and then use that file for both.

Reply:
I tried to use the existing CMake file for the super build, but it tried to compile everything from the source, and it failed to compile protobuf.

So for now, maybe we can use the separate CMake file and consolidate them later?

Reply:
SGTM.